lib/services/platform-service.ts

```diff
@@ -256,18 +256,18 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		return true;
 	}
 
-	public async validateOptions(provision: any, projectData: IProjectData, platform?: string): Promise<boolean> {
+	public async validateOptions(provision: true | string, projectData: IProjectData, platform?: string): Promise<boolean> {
```

Comment:
I believe the type was any as there's a chance to get here with object, not boolean or string value (at least in the current code in master branch). For example [here](https://github.com/NativeScript/nativescript-cli/blob/ddd8328a30d33e997d975107b58c240d0b0c9a59/lib/commands/build.ts#L49) - in case you call:
```
$ tns build ios --provision.prop1
```
here you'll receive an object: ` { prop1: true }`
Maybe we should guard the `$options.provision`, but I agree to handle this later (separate PR) as it is not expected to pass such values to `--provision`.